all-programs-summary-new/allprograms.json

```diff
@@ -0,0 +1,1718 @@
+[
+  {
+    "State": "AK",
+    "Title I 2018": 145776.0,
+    "Title I 2019": 124161.0,
+    "Title I 2020": 108899.0,
+    "Title I 2021": 0.0,
+    "Title I 2022": 0.0,
+    "Title I Total": 378836.0,
+    "Title II 2018": 7662394.39,
+    "Title II 2019": 6729781.45,
+    "Title II 2020": 8570223.77,
+    "Title II 2021": 9606129.47,
+    "Title II 2022": 8893772.0,
+    "Title II Total": 41462301.08,
+    "Crop Insurance 2018": 28332677.0,
```

Comment:
Crop insurance doesn't look right. If I am reading this right, Alaska shows 28,332,677 for 2018, but our current map hover shows 81,000

Reply:
Yes, there was a mix up when importing all program numbers into the All Programs (Final) sheet from Crop Insurance (Main) sheet. Only the data for crop insurance in those two states were affected. The grand totals doesn't change too. I have fixed the push in data-import PR and this PR. Please verify and resolve this conversation, if it looks good. Thanks for finding this.